arm-recoveryservices/2016-06-01/swagger/registeredidentities.json

```diff
@@ -0,0 +1,388 @@
+    "application/json"
+  ],
+  "paths": {
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/certificates/{certificateName}": {
+      "put": {
+        "tags": [
+          "VaultCertificates"
+        ],
+        "description": "Upload a certificate for a resource.",
+        "operationId": "VaultCertificates_Upload",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "name": "certificateName",
+            "in": "path",
+            "description": "Certificate friendly name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "certificateRequest",
+            "in": "body",
+            "description": "Input parameters for uploading the vault certificate.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CertificateRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ResourceCertificateDetailsResource"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/registeredIdentities/{identityName}": {
+      "delete": {
+        "tags": [
+          "RegisteredIdentities"
+        ],
+        "description": "Unregisters the given container from your Recovery Services vault.",
+        "operationId": "RegisteredIdentities_Delete",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "name": "identityName",
+            "in": "path",
+            "description": "Name of the protection container to unregister.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "NoContent"
+          }
+        },
+        "deprecated": false
+      }
+    }
+  },
+  "definitions": {
+    "CertificateRequest": {
+      "description": "Details of the certificate to be uploaded to the vault.",
+      "type": "object",
+      "properties": {
+        "properties": {

```

Comment:
This is a polymorphic response type and we observed previously that this structure needs to be preserved for polymorphism to work in the intended manner.